deps/prebuilt/include/TAEF/WexTestClass.h

```diff
@@ -115,13 +119,44 @@ extern "C" __declspec(noreturn) void __fastfail(unsigned int);
 #define TAEF_FAST_FAIL System::Environment::FailFast(nullptr)
 #endif // #if !defined(__cplusplus_cli)
 
+#if defined(__clang__)
```

Comment:
yes agreed. Was just putting it close to the use as I was trying things out.

---
In reply to: [92518736](https://github.com/Microsoft/WinObjC/pull/1523#discussion_r92518736) [](ancestors = 92518736)